docs/site/paredit.md

```diff
@@ -101,6 +101,7 @@ Default keybinding                | Action | Description
  `ctrl+backspace`                    | **Kill List Backward** | Deletes everything from the cursor to the opening of the current enclosing form.<br> ![](images/paredit/kill-open-list.gif)
  `ctrl+alt+shift+delete`                 | **Splice Killing Forward** | Delete forward to end of the list, then Splice. <br> ![](images/paredit/splice-killing-forward.gif)
  `ctrl+alt+shift+backspace`              | **Splice Killing Backwards** | Delete backward to the start of the list, then Splice. <br> ![](images/paredit/splice-killing-backward.gif)
+ `ctrl+k` | **Kill Right** | Delete forward to the end of the current form or the first newline.<br> ![](images/paredit/kill-right.gif)
```

Comment:
Need an image for `kill-right.gif`, but not sure how to.

Reply:
I use GIF Brewery. Let me know if you want me to record it. I think it I can set it up pretty easily.

Reply:
@PEZ If you wouldn't mind, I'd greatly appreciate it! Thank-you.